RELEASENOTES.md

```diff
@@ -41,6 +41,77 @@ repository but is still accessible under the
 
 ---
 
+### Networking Recipe
+
+The networking recipe enhances and extends the functionality enabled in 23.01
+release.
+
+#### Features and enhancements
+
+* Intel IPU E2100 target support
+* IPsec offload support (Intel IPU E2100 target only)
+* IPv6 support in infrap4d networking stack (Intel IPU E2100 target only)
+* Local mirroring (Intel IPU E2100 target only)
+* Fedora 37 support
+* ACC support for Rocky Linux 9.1 container
+* Cross-compile for ARM64 Rocky Linux 9.1
+* Multi-process TDI stack support (ability to run infrap4d as secondary process
+in multiple control planes environment)
+* Support for remote gRPC clients (infrap4d on ACC and gRPC clients on host)
+* Unit test framework
+* Security enhancements
+  * Introduction of `sgnmi_cli`, a secure-by-default
+gNMI client
+  * Library updates to address CVE security issues
```

Comment:
Should we say grpc and c-ares library updates?

Reply:
We dont have the grpc CVE issues addressed yet (the PR is open and needs testing).
Update to include the c-ares CVEs